Fix "Apply now" button on join-us page

AlexARNcode opened the pull request:
## Purpose
The purpose of this PR is to fix #756

## Goals
We need to make both buttons the same styling, which is not the case right now.

## Approach
I'm using this as a guideline : https://stackoverflow.com/questions/2906582/how-to-create-an-html-button-that-acts-like-a-link
I'm nesting a "div" with the button css style, inside a "a".

### Screenshots

  
### Preview Link
<!---  This PR will be automatically deployed to surge. -->
<!---  Once you submit the PR, replace "{PR_NUMBER}" with your PR number. -->
https://pr-760-sef-site.surge.sh/join-us.html

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
<!--- List any other related PRs --> 

## Test environment
<!--- List all JDK versions, operating systems, databases, and browser/versions on which this feature/fix was tested --> 

## Learning
I tried to use good practices for buttons, see : https://stackoverflow.com/questions/2906582/how-to-create-an-html-button-that-acts-like-a-link

AlexARNcode committed:
fix apply now button on join-us.html page

Piumal1999 commented on src/main/webapp/join-us.html:
```diff
@@ -203,11 +203,7 @@ <h5 class="mb-0">
-                    >
-                        Apply now!
-                    </a>
+                    <a href="apply.html"><button class="btn btn-purple text-uppercase mt-3" href="apply.html">Apply now!</button></a>
```
It's better if we can indent the code like this  -->
```
<a href="apply.html">
     <button class="btn btn-purple text-uppercase mt-3">
          Apply now!
     </button>
</a>
```

Piumal1999 replied:
Also we can remove the `href` attribute in the button tag

AlexARNcode replied:
No problem, I'll do that :) !
Actually, it looks like a button shouldn't be nested in a "a href". If you try to validate the HTML code in https://validator.w3.org for example, it will tell you that.

AlexARNcode replied:
> Also we can remove the `href` attribute in the button tag

Yep, already did for this button, but we should do it for the other indeed, and/or do what I told you in my previous comment ?

Piumal1999 replied:
Oh right. Then let's try adding `!important` rule for the color of "btn-purple" class. So we can keep the join-us.html file same as before

AlexARNcode replied:
Hi mate,
What I meant is this : https://stackoverflow.com/questions/2906582/how-to-create-an-html-button-that-acts-like-a-link

Either the HTML way (with form) or CSS way (create a "a" styled as a button with css). Tell me what you preefer and I'll do it ;)

Gravewalker666 replied:
Hello @AlexARNcode ,
Let's keep only the button tag and place the href as a prop of the button instead of wrapping the button in an `<a>` or adding CSS to the `<a>` to look like button.

AlexARNcode committed:
fix apply now button on join-us.html page v2

Piumal1999 reviewed (changes requested):
Well done @AlexARNcode !!! We can merge this after some small changes. I have added some comments. If you have any questions, feel free to ask via gitter or sef hive(https://sef.discourse.group)

And could you please copy and fill the pull request template so it will be easy for us to review. 
Link: https://github.com/sef-global/sef-site/blob/master/pull_request_template.md

AlexARNcode committed:
Buttons styling on join-us.html page

Gravewalker666 reviewed (changes requested):
@AlexARNcode Thanks for keeping the conversation alive mate! 
Please fill out the generated PR template form so that we all can have a better understanding of what this PR does and how you did it. So that even after merging the PR we can get an idea of what happened to the code and how and why the changes were made.

AlexARNcode committed:
Buttons styling on join-us.html page v2

AlexARNcode committed:
join-us.html page : Styling buttons v2

I'm trying to style the buttons of the join-us.html page following good html/css practice.

AlexARNcode reviewed (commented):
Forget this commit, I'm not a github ninja yet, as you can see.. Sorry for that.

Gravewalker666 reviewed (approved):
@AlexARNcode Good job dude, well done!!!

Gimhan-minion reviewed (approved):
Thank you!
@AlexARNcode :100: :heart:
